src/Components/Publishing/Header2/Layouts/Components/FeatureBasicHeader.tsx

```diff
@@ -0,0 +1,147 @@
+import { space } from "@artsy/palette"
+import { BylineContainer } from "Components/Publishing/Byline/Byline"
+import { ShareContainer } from "Components/Publishing/Byline/Share"
+import {
+  CoverImage,
+  IFrame,
+  isValidVideoUrl,
+  Video,
+  VIDEO_RATIO,
+} from "Components/Publishing/Sections/Video"
+import React from "react"
+import styled from "styled-components"
+import { track } from "../../../../../Utils/track"
+import { pMedia } from "../../../../Helpers"
+import { EditImage, FeatureHeaderProps } from "../FeatureHeader"
+import {
+  Deck,
+  FeatureInnerContent,
+  SubContentContainer,
+  Title,
+} from "./FeatureInnerContent"
+
+interface State {
+  isPlaying: boolean
+}
+
+export class FeatureBasicHeader extends React.Component<
+  FeatureHeaderProps & { tracking?: any },
+  State
+> {
+  trackVideoPlay = () => {
```

Comment:
Based on our discussion about analytics and types. We should at least use the Tracking schema here.
https://github.com/artsy/reaction/pull/994/files#diff-b7e2851428e5c7793760ae30da591015